keyboards/kudox/rev1/rules.mk

```diff
@@ -0,0 +1 @@
+BACKLIGHT_ENABLE = no
```

Comment:
```suggestion
```

`BACKLIGHT_ENABLE` is already set to `no` in the keyboard-level rules.mk, so this isn't necessary.